src/app/components/Logout.tsx

```diff
@@ -0,0 +1,8 @@
+"use client"
+import { Button, Typography } from "@mui/material";
+import { signOut } from "next-auth/react";
+export default function Logout() {
+  return <Button onClick={() => signOut()} variant="contained" color="secondary">
+    <Typography>Sign out</Typography>
+  </Button>
+}
```

Comment:
minor: Lets add some new line separators between the main sections

```suggestion
"use client";

import { Button, Typography } from "@mui/material";
import { signOut } from "next-auth/react";

export default function Logout() {
  return (
    <Button 
      onClick={() => signOut()} 
      variant="contained" 
      color="secondary"
    >
      <Typography>Sign out</Typography>
    </Button>
  );
}
```